app/controllers/internal/broadcasts_controller.rb

```diff
@@ -36,6 +36,17 @@ def index
                   end.order(title: :asc)
   end
 
+  def destroy
+    broadcast = Broadcast.find_by(id: params[:id])
```

Comment:
```suggestion
    broadcast = Broadcast.find_by!(id: params[:id])
```
To ensure that if the broadcast can't be found, we raise an error rather than try to call `destroy` on something that is `nil` at that point :)

Reply:
I removed the exclamation point when I changed the method back. 🤦‍♀️  Making the change now!